src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -1118,6 +1118,30 @@ class LinearIPRDescription(CommonIPR):
     # fmt: on
     # [[[end]]] (checksum: 433953e29d06e22612c935acdbd70db4)
 
+@attr.s(frozen=True, slots=True)
+class VogelIPRDescription(CommonIPR):
+    """
+    .. include:: /alfacase_definitions/VogelIPRDescription.txt
+
+    .. include:: /alfacase_definitions/list_of_unit_for_pressure.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_standard_volume_per_time.txt
+    """
+
+    min_pressure_difference = attrib_scalar(default=Scalar(0.0, "Pa"))
+
+    # [[[cog
+    # cog_out_multi_input("well_max_flow_rate", "standard volume per time", 0.0, "sm3/d")
+    # ]]]
+    # fmt: off
+    well_max_flow_rate_input_type = attrib_enum(default=constants.MultiInputType.Constant)
+    well_max_flow_rate = attrib_scalar(
+        default=Scalar('standard volume per time', 0.0, 'sm3/d')
```

Comment:
According to Vogel's model, the Max Flow Rate is used to calculate the Dimensionless Flow Rate (q/q_max). I suggest using a value different from zero for a default value. Perhaps `1.0`. 
What do you think? Does it make sense?

Reply:
Correct, the Vogel's deduction considers a dimensionless flow rate. But, we ended up computing the flow rate as a function of q_max, P_well and P_res, in other words:
```
q = q_max* (1 - 0.2 * (P_well/P_res) - 0.8 * (P_well/P_res)^2)
``` 
So, q_max is no longer in the denominator of a fraction, which means no division by zero.
However, to be consistent with Vogel's approach, using q_max = 1.0 by default is not a bad idea.

Reply:
You are right! I forgot about it. Since we want to have any flow rate in the IPR, I think keeping the default value equal to one  seems to be a good approach.